### PR TITLE
Add announcement for OMERO.web 5.20.0

### PR DESCRIPTION
**Pull request by knabar**
.

### CODE REVIEW

**Comment by jburel** [comment]
Maybe add a link to security/advisories/2023-31047/index.html

File: _posts/2023-05-10-omero-web-5.20.0.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: Release of OMERO.web 5.20.0
+intro-blurb: The OME team is pleased to announce the release of OMERO.web 5.20.0
+---
+
+Today we are releasing OMERO.web 5.20.0, which includes a fix for multi-file upload validation, 
+support for the [Django 3.2.19 security update](https://www.openmicroscopy.org/security/advisories/2023-31047/index.html),
```
Comment (jburel) [inline _posts/2023-05-10-omero-web-5.20.0.md:8]: ``{{ site.baseurl }}/security/advisories/2023-31047/`` should be used

**Comment by sbesson** [comment]
@knabar can we also update https://www.openmicroscopy.org/security/advisories/2023-31047/ to mention the resolution in the OMERO.web release?

**Comment by jburel** [comment]
https://snoopycrimecop.github.io/www.openmicroscopy.org/2023/05/10/omero-web-5.20.0.html looks good

**Comment by jburel** [comment]
The synopsis of ``security/advisories/2023-31047/`` needs to be adjusted i.e.
``Django security release 3.2.19 is incompatible with latest OMERO.web``
Also the url is not formatted correctly see https://snoopycrimecop.github.io/www.openmicroscopy.org/security/advisories/2023-31047/

**Comment by jburel** [comment]
https://snoopycrimecop.github.io/www.openmicroscopy.org/security/advisories/2023-31047/ looks good now
Github is now stable at the moment see https://www.githubstatus.com/
So we should wait a bit before merging and tagging

File: _posts/2023-05-10-omero-web-5.20.0.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+### Installing the Software
+
+OMERO.web 5.20.0 is available from PyPI - see 
+[here](https://pypi.org/project/omero-web/5.20.0/).
+
+Official Docker images are available as usual on Docker Hub with either
+the latest or the 5.20 tag:
+
+* <https://hub.docker.com/r/openmicroscopy/omero-web-standalone>
+* <https://hub.docker.com/r/openmicroscopy/omero-server>
```
Comment (sbesson) [inline _posts/2023-05-10-omero-web-5.20.0.md:22]: Should this be https://hub.docker.com/r/openmicroscopy/omero-web ?

**Review by sbesson — COMMENTED** [review]
Apart from one comment about Docker images, announcement of advisory updates look good to me

File: _posts/2023-05-10-omero-web-5.20.0.md (hunk 2 above)
Comment (jburel) [reply _posts/2023-05-10-omero-web-5.20.0.md:22]: it should

**Review by sbesson — APPROVED** [review]
Thanks Andreas

**Comment by jburel** [comment]
thanks Merging and tagging

**Pull request merged**